mock-hpos-api/defaultResponse.js

```diff
@@ -0,0 +1,56 @@
+const _ = require('lodash')
+
+// mock of normal hpos api responses
+
+const happs = [{
+  id: 1,
+  name: 'HoloFuel'
+}, {
+  id: 2,
+  name: 'Community'
+}, {
+  id: 3,
+  name: 'Elemental Chat'
+}]
+
+
+const data = {
+  get: {   
+    '/hosted_happs': {
+      hosted_happs: happs
+    },
+    '/config': {
+      admin: {
+        public_key: '5m5srup6m3b2iilrsqmxu6ydp8p8cr0rdbh4wamupk3s4sxqr5',
+        email: 'test@test.com'
+      },
+      holoportos: {
```

Comment:
GET `/config` returns the "settings"  sub-object of the `hpos-config.json` file. The default information is just:
```
{
  "admin": {
    "public_key": "my_holoport_key",
    "email": "my_email"
  }
}

But you can PUT any arbitrary information there (e.g. name).

That being said, neither `holoportos.sshAccess` nor `holoportos.network` are used in any way. 

To get SSH status:
GET `/profiles/development/features/ssh` should return something like `"enabled": true`

Please double check the exact format of the response. I'm not great with parsing it in my head: https://github.com/Holo-Host/holo-nixpkgs/blob/7226c977c9eb44eb440a4d135f088c30ae7dcd3e/overlays/holo-nixpkgs/hpos-admin-api/hpos-admin-api.py#L123-L130

Reply:
Yeah, that was a mistake. I read an outdated part of the rfc, but missed the up to date part. Afaict you're right about the response format.